iOS/README.md

```diff
@@ -36,7 +36,10 @@ To try out the Ping iOS SDK please look at one of our samples:
 
 ### Integrate with PingOne:
 
-- Samples coming soon
+-  [**Davinci using SwiftUI - `/ios/swiftui-davinci`**](./swiftui-davinci/)
```

Comment:
Add a README file to the sample app folder. You can follow the structure of the `kotlin-davinci`